### PR TITLE
[Fix] Fix auto scale lr

### PR DESCRIPTION
**Pull request by chhluo**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by jbwang1997** [comment]
Some configs like yolox need to uncomment auto_scale_lr settings

File: configs/_base_/default_runtime.py [hunk 1]
```diff
@@ -22,3 +22,9 @@
 log_level = 'INFO'
 load_from = None
 resume = False
+
+# Default setting for scaling LR automatically
+#   - `enable` means enable scaling LR automatically
+#       or not by default.
+#   - `base_batch_size` = (8 GPUs) x (2 samples per GPU).
+auto_scale_lr = dict(enable=False, base_batch_size=16)
```
Comment (RangiLyu) [inline configs/_base_/default_runtime.py:30]: This should not be put in default_runtime. May put it in the schedule folder.

**Review by jbwang1997 — APPROVED** [review]
LGTM

File: tools/train.py [hunk 2]
```diff
@@ -81,6 +86,18 @@ def main():
...
             cfg.optim_wrapper.loss_scale = 'dynamic'
 
+    # enable automatically scaling LR
+    if args.auto_scale_lr:
+        if 'auto_scale_lr' in cfg and \
+                'enable' in cfg.auto_scale_lr and \
+                'base_batch_size' in cfg.auto_scale_lr:
+            cfg.auto_scale_lr.enable = True
+        else:
+            warnings.warn('Can not find "auto_scale_lr" or '
```
Comment (RangiLyu) [inline tools/train.py:96]: Should we raise an error instead of a warning here?

**Review by RangiLyu — APPROVED** [review]

**Pull request merged**